### DISCUSSION TITLE
Sudo Problem (macOS Ventura)

### DISCUSSION THREAD

**Discussion by yewonyoana** [Homebrew/discussions#4756] [Getting started] [answered]
### Output of `brew config`

```shell
❯ /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
==> Checking for `sudo` access (which may request your password)...
Password:
```

### Output of `brew doctor`

```shell
I can't even run this because I can't install it.
```

### Description of issue

Hi.
So I tried installing Homebrew, the pkg way and terminal way, but bumped into problems both ways.

Homebrew pkg does install on my iMac 2020, macOS Ventura.
But when I tried to add MongoDB, it asked for "sudo" access again.

As for the terminal way, it's like the above: checking for "sudo" access.
I type in the password, and the below image pops up.

I tried installing Homebrew sudoless way, but my iMac was really about to explode.

I tried all sorts by Googling and stuff, and nothing worked.
And yes, my account is the admin.

Can someone please help me?
Thank you!

<img width="782" alt="Screenshot 2023-09-01 at 9 34 20 AM" src="https://github.com/Homebrew/discussions/assets/115424953/6414d1b0-7900-40ad-9370-34f4c3f8d61c">

**Comment by SMillerDev** [comment, marked as answer]
What happens when you do `sudo echo "test"`?

**Reply by yewonyoana** [reply]
It again asks for the password, and the same screen as above pops up :(

**Reply by yewonyoana** [reply]
Hi! I was actually able to solve it somehow. But thank you so much for the reply!
